content/neuvector-prime/azure/contents.lr

```diff
@@ -11,89 +11,76 @@ A step by step walk-through for creating a new offer from the Marketplace Overvi
 1. Choose the plan from the dropdown list. ( "Plans + Pricing" tab more details about Software plan)
 2. Click 'Create' 
 
-## Basic (Project Details)
+## Basic
 
 1. Select existing subscription from the dropdown list.
-2. Select existing Resource group from the dropdown list or create new Resource group.
+2. Select existing Resource group from the dropdown list.
 
    Existing Resource Group selection:
 
    !["Existng Resource Group"](./existingrgclean.png)
 
-   Create New Resource group:
-   Provide name for new Resource group.
 
-   !["New Resource Group"](./newresourcegroupclean.png)
 
-3. Choose New cluster [ Yes/No ] option. 
 
-   Yes option
+   ```
+   Note: Create New Resource Group feature is not supported for Neuvector and 
+         Configure Billing-adapter offer
+   ```
 
-   Choose Region
 
-   !["Create Cluster"](./createclusteryesclean.png)
 
-   click 'Next'
+   !["New Resource Group"](./createnewresourcegroup.png)
 
-    Cluster Details:
 
-     1. Choose AKS Cluster Name
-     2. Choose Kubernetes Version from the dropdown list
-     3. Choose VM size
-     4. Enable auto scaling    
-     5. Slide the bar to choose Virtual Machine count
-     
-     !["Cluster Details"](./clusterdetailsyes.png)
 
+## Cluster Details
 
+   Choose Cluster name from dropdown list.
 
-   No option
-  
-   click 'Next'
 
-   Cluster Details:
 
-   Choose AKS Cluster name from dropdown list
 
-   !["Create Cluster"](./clusterdetailsno.png)
+   !["Cluster details"](./clusterdetailsno.png)
 
 
 
 
    click 'Next'
 
+
 ## Application Details
 
+
+
 1. Choose name for Cluster extension resource name
 2. Choose name for Application Title 
 
 
 !["Application Details"](./application_details.png)
 
 click 'Next'
+
+
 ## Neuvector Configuration
 
+
 1. Choose Admin password 
 2. Confirm Admin password 
 
 click 'Next'
 
 ## Review + create
 
-It will summarize the offer and also view automation template 
-
-Price:
-  Plan 
-  Price 
-  Terms
-
-Basics:
 
-Cluster Details:
+  It will summarize the offer and also view automation template 
```

Comment:
s/view/display/

Reply:
this should be "view automation template"

I changed the above to 

It will summarize the offer and link to "view automation template" (Azure Resource Manager Template)